msteams-platform/concepts/device-capabilities/location-capability.md

```diff
@@ -10,54 +10,58 @@ ms.author: surbhigupta
+The main advantage of integrating location capabilities in your Teams apps is that it allows web, desktop, and mobile app developers on Teams platform to leverage location functionality with Microsoft Teams JavaScript client SDK.
+
+The following examples show how the integration of location capabilities is used in different scenarios:
 
-* In a factory, the supervisor can track the attendance of workers by asking them to take a selfie in the vicinity of the factory and share it through the specified app. The location data also gets captured and sent along with the image.
-* The location capabilities enables the maintenance staff of a service provider to share authentic health data of cellular towers with the management. The management can compare any mismatch between captured location information and the data submitted by maintenance staff.
+* Track the attendance by using a selfie in the vicinity of the area. The location data also gets captured and sent along with the image.
+* Share authentic health data of cellular towers with the management. The management can compare any mismatch between captured location information and the data submitted by maintenance staff.
+* Locate technical support staff in a particular area. The app asks for permission to use the location to find support staffs near the specified area. After the permission is granted, the search results are filtered near that particular location and support staff IDs get displayed.
+* Report the location after completing a job in the field.
 
-To integrate location capabilities, you must update the app manifest file and call the APIs. For effective integration, you must have a good understanding of [code snippets](#code-snippets) for calling the location APIs.
-It is important to familiarize yourself with the [API response errors](#error-handling) to handle the errors in your Teams app.
+To integrate location capabilities, you must do the following:
 
-> [!NOTE]
-> Currently, Microsoft Teams support for location capabilities is available for mobile clients only.
+* Update the app manifest file and call the APIs.
+* Have working knowledge of [code snippets](#code-snippets) for calling the location APIs.
+* Handle errors in your Teams app with the help of [API response errors](#error-handling).
 
 ## Update manifest
 
-Update your Teams app [manifest.json](../../resources/schema/manifest-schema.md#devicepermissions) file by adding the `devicePermissions` property and specifying `geolocation`. It allows your app to ask for requisite permissions from users before they start using the location capabilities. The update for app manifest is as follows:
+Update your Teams app [manifest.json](../../resources/schema/manifest-schema.md#devicepermissions) file by adding the `devicePermissions` property and specifying `geolocation`. It allows your app to ask for requisite permissions from users before they start using the location  capabilities
```

Comment:
It allows your app to ask for required permissions from users before they start using the location capabilities.